haskell/test/Test/SegCFMM/Contract.hs

```diff
@@ -14,4 +14,4 @@ import Util (fetchContract)
 
 segCFMMContract :: Contract (ToT Parameter) (ToT Storage)
 segCFMMContract =
- $(fetchContract @(ToT Parameter) @(ToT Storage) "SEGMENTED_CFMM_PATH")
+ $(fetchContract @Parameter @Storage "SEGMENTED_CFMM_PATH")
```

Comment:
In this new version of `morley` there is a utility that can replace `fetchContract`, see `embedContract`  in `Test.Cleveland.Lorentz.Import`.

Note: it should be possible to remove other utility functions here as a consequence.

Reply:
Maybe you looked the old version of the file, current it is already updated to use `embedContractM`:

```
segCFMMContract :: Contract (Parameter) (Storage)
segCFMMContract =
  $$(embedContractM @Parameter @Storage
      (resolveSourcePath "haskell/test/segmented_cfmm_default.tz" "SEGMENTED_CFMM_PATH"))
```

Reply:
Oh probably, sorry about that.

Why not `embedContract` with a static path however?

Reply:
Hmm maybe using the env var will be useful in the future? With the changes from previous PR, I can remove the env var and just the static path if it is no longer useful.

Reply:
Yes, it seems it's no longer needed and that we can remove it.

I think leaving it now would only be confusing, if we need it later we can add it back.

Reply:
Yes, it seems it's no longer needed and that we can remove it.

I think leaving it now would only be confusing, if we need it later we can add it back.

Reply:
Make sense, I've removed this.